ti/instrumentation/utils.go

```diff
@@ -196,31 +197,47 @@ func createConfigFile(runner TestRunner, packages, annotations, workspace, tmpDi
 		}
 		packages = strings.Join(pkgs, ",")
 	}
+	var data string
+	var outputFile string
 
-	data := fmt.Sprintf(`outDir: %s
+	if !yaml {
+		outputFile = fmt.Sprintf("%s/config.ini", tmpDir)
+		data = fmt.Sprintf(`outDir: %s
 logLevel: 0
 logConsole: false
 writeTo: COVERAGE_JSON
 instrPackages: %s`, dir, packages)
+	} else {
+		outputFile = fmt.Sprintf("%s/Config.yaml", tmpDir)
+		p := strings.Split(packages, ",")
+		for idx, s := range p {
+			p[idx] = fmt.Sprintf("'%s'", s)
+		}
+		data = fmt.Sprintf(`outDir: '%s'
```

Comment:
Use golang struct and then yaml marshal to generate the data for yml file https://dev.to/sagartrimukhe/generate-yaml-files-in-golang-29h1

Reply:
I have added a TODO for this once we use yaml consistently for all the languages